<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="dateBox">
         <property name="title">
          <string>Период времени</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="initLabel">
            <property name="text">
             <string>От</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="outLabel">
            <property name="text">
             <string>До</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDateEdit" name="outDate">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::DaySection</enum>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="initialDate">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>initLabel</zorder>
         <zorder>initialDate</zorder>
         <zorder>outLabel</zorder>
         <zorder>outDate</zorder>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="investBox">
         <property name="title">
          <string>Начальные инвестиции</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="beginInvest">
            <property name="minimum">
             <double>90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableWidget" name="packagesTable">
      <property name="maximumSize">
       <size>
        <width>205</width>
        <height>16777215</height>
       </size>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Case</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Profit</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTableWidget" name="resultTable">
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>In</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Out</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cash</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>90</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>270</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>810</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2410</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>7230</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>21690</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>65610</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
